src/OMSimulatorLib/Scope.cpp

```diff
@@ -297,20 +294,22 @@ oms_status_enu_t oms3::Scope::getElement(const oms3::ComRef& cref, oms3::Element
     return logError("A model has no element information");
 
   oms3::System* system = model->getSystem(tail);
-  if (!system)
-    return logError("Model \"" + std::string(front) + "\" does not contain system \"" + std::string(tail) + "\"");
+  oms3::Component* component = model->getComponent(tail);
+  if (!system && !element)
```

Comment:
I think `element` is a typo here.

```suggestion
  if (!system && !component)
```